doc/source/getting_started/wsl.rst

```diff
@@ -213,8 +218,48 @@ If you want to change the working directory, you can use the ``-dir`` flag.
     /ansys_inc/v222/ansys/bin/ansys222 -grpc -dir /tmp/ansys_jobs/myjob
 
 
+Launch MAPDL in the Windows host OS
+###################################
+
+You can launch an instance of MAPDL using the MAPDL installation from the
+Windows host OS.
+To do that, you can just run the following code.
+
+.. code:: python
+
+   from ansys.mapdl.core import launch_mapdl
+
+   mapdl = launch_mapdl(
+       exec_file="/mnt/c/Program Files/ANSYS Inc/v231/ANSYS/bin/winx64/ANSYS231.exe",
+   )
+
+As mentioned in `Open ports for license server communication`_, the Windows host OS
+and WSL are connected with a virtual network where they have both different IPs.
```

Comment:
```suggestion
and WSL are connected with a virtual network where they both have different IP addresses.
```
According to Google developer documentation style guide, we should always use IP address and not just IP as it has other meanings.